UI.PageModels/Pages/Admin/Booking/ParticipantsPage.cs

```diff
@@ -154,4 +158,17 @@ public VideoAccessPointsPage GoToVideoAccessPointsPage()
         ClickElement(_nextButton);
         return new VideoAccessPointsPage(Driver, DefaultWaitTime);
     }
+    
+    public SummaryPage GoToSummaryPage()
```

Comment:
When would a use want to go to the SummaryPage from the participant page in the journey in your test?

Reply:
there is a summary page to drive participant to next

Reply:
var summaryPage = addParticipantPage.GoToSummaryPage();

Reply:
please let have chat when ever you are free